decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -146,28 +146,27 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
     
     @IBAction func onConfirm() {
         if self.userEnteredSeedWords.contains("") {
-            self.displaySeedError("Not all seeds are entered. Please, check input fields and enter all seeds.")
-        }
-        else{
+            self.invalidSeedLabel("Not all seeds are entered. Please, check input fields and enter all seeds.")
+        }else{
             let validatedSeed = self.validateSeed()
             if validatedSeed.valid {
                 self.secureWallet(validatedSeed.seed)
             }
             else {
-                self.displaySeedError("You entered an incorrect seed. Please check your words.")
+                self.invalidSeedLabel("You entered an incorrect seed. Please check your words.")
             }
         }
     }
     
-    func displaySeedError(_ errorMessage: String) {
-        self.lblEnterAllSeeds.isHidden = false
+    func invalidSeedLabel(_ errorMessage: String) {
```

Comment:
`displaySeedError` is a fine function name, I meant to rename the `lblEnterAllSeeds` label.